src/benchmarks/src/bench_utils.cpp

```diff
@@ -0,0 +1,174 @@
+#include <clipper/json_util.hpp>
+#include <clipper/logging.hpp>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <unordered_map>
+
+#include "include/bench_utils.hpp"
+
+using namespace clipper;
+
+namespace bench_utils {
+
+const std::string CIFAR_DATA_PATH_PROMPT =
```

Comment:
Only a handful of these (and their counterpart variables here https://github.com/ucbrise/clipper/pull/225/files#diff-5ac4406c77f36f86be1d3ed9ab6795fbR8) are used by `end_to_end_bench`. The rest are used by the throughput benchmarking script that isn't included. Leaving these strings around might reduce busywork in the future, but it also could just add noise. Happy to remove if that seems right.